client/src/Movies/MovieCard.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+
```

Comment:
You could have pushed the data from Movie.js to the card instead. If you ever wanted too to change this around in the future.